### PR TITLE
Update knexfile.js

### PR DESCRIPTION
**Pull request by shravan1k**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
Added dotenv configuration in knexfile.js.

### Testing
How did you confirm your changes worked? 
- Before the change, trying to migrate to the database gave an error. After the change, the migration was successful.

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

Screen Shot 2020-12-03 at 12 02 59 PM

| [TODO Screenshot Title] |
| --- |
| [TODO Screenshot Link] |

### CODE REVIEW

File: knexfile.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Update with your config settings.
```
Comment (mingkaili) [inline knexfile.js]: Can you delete this comment right here? I think I might have accidentally put this comment here.

Comment (shravan1k) [reply knexfile.js]: Done.

**Review by mingkaili — APPROVED** [review]
LGTM

**Review by eliasfang — APPROVED** [review]
Nice! LGTM

**Pull request merged**